### ISSUE TITLE
Missing focus indicator for social media sharing icons in book header

### ISSUE THREAD

**Issue by MichSpieler** [pressbooks/pressbooks#4429]
The visual focus indicator (VFI) doesn’t highlight the social media icons at the bottom of books. See example below from[ https://openfl.pressbooks.pub/man4162](https://openfl.pressbooks.pub/man4162)

<img width="1072" height="424" alt="Image" src="https://github.com/user-attachments/assets/17423339-7a73-4606-b4b7-b4c9fd45fe59" />

**Comment by MichSpieler** [comment]
The sharing buttons are not reachable by keyboard.

**Comment by tw77** [comment]
Could be tackled along with https://github.com/pressbooks/private/issues/2397

**Comment by arzola** [comment]
https://github.com/pressbooks/pressbooks-book/pull/1439

**Comment by tw77** [comment]
Fix works as expected on dev, ready for release
